config/collector.yaml

```diff
@@ -64,6 +66,19 @@ processors:
     limit_percentage: 65
     spike_limit_percentage: 20
 
+  metricstransform/self-metrics:
+    transforms:
+    - action: update
+      include: otelcol_process_uptime
+      operations:
+      - action: add_label
+        new_label: version
+        new_value: Google-Cloud-OTLP manifests:0.1.0 otel/opentelemetry-collector-contrib:0.99.0
+      - action: aggregate_labels
```

Comment:
Is this metric positioned as an "internal" one that we use for troubleshooting like we do in the Ops Agent? Or is it a self-metric that's intentionally customer-visible?

If it's the latter, are we losing any valuable (to the customer) labels with this operation?

Reply:
We are working on a dashboard for these self-metrics, the other labels here are:

- service.instance.id: I don't think this is particularly valuable to customers
- service.scope.name: otel/prometheusreceiver -> if anything this is just an implementation detail
- otel_scope and otel_version

I think its fine to aggregate these away but I think we can remove the aggregations for now, and instead if we want these aggregated we can do that in our dashboard samples instead of in the collector.

I'll push that change shortly